lib/LaTeXML/Core/Rewrite.pm

```diff
@@ -438,6 +439,19 @@ sub domToXPath_rec {
         return ($xpath, $n, $n); }
       else {
         return ($axis . '::*', 1, 1); } }
+    # Also, an XMRef pointing to a wildcard is a wildcard!
+    # (or pointing to an XMArg|XMWrap of a wildcard!)
+    elsif ($qname eq 'ltx:XMRef') {
+      my $id = $node->getAttribute('idref');
```

Comment:
Another tiny question, can't we use the `->realizeXMNode` here as well? May be helpful if XMRef handling is refactored one day.

Reply:
That was my first thought when writing, but you don't actually want to do wildcard handling on the referenced node (I think).